docs/onboarding/installation.md

```diff
@@ -29,17 +29,17 @@ ironfish
 
 <br />
 
-When new versions are announced, you can update through NPM:
```

Comment:
I think the grammar is more correct as it is -- here `you can update through NPM` means that you are updating your installed Iron Fish. Changing to `you can update them` seems to suggest that you are updating the new versions themselves.